build.sbt

```diff
@@ -36,41 +36,41 @@ def isScala213: Boolean = System.getProperty("akka.build.scalaVersion", "").star
 
 // When this is updated the set of modules in ActorSystem.allModules should also be updated
 lazy val aggregatedProjects: Seq[ProjectReference] = List[ProjectReference](
-    actor,
-    actorTests,
-    actorTestkitTyped,
-    actorTyped,
-    actorTypedTests,
-    benchJmh,
-    benchJmhTyped,
-    cluster,
-    clusterMetrics,
-    clusterSharding,
-    clusterShardingTyped,
-    clusterTools,
-    clusterTyped,
-    coordination,
-    discovery,
-    distributedData,
-    docs,
-    jackson,
-    multiNodeTestkit,
-    osgi,
-    persistence,
-    persistenceQuery,
-    persistenceShared,
-    persistenceTck,
-    persistenceTyped,
-    protobuf,
-    remote,
-    remoteTests,
-    slf4j,
-    stream,
-    streamTestkit,
-    streamTests,
-    streamTestsTck,
-    streamTyped,
-    testkit)
+  actor,
```

Comment:
intended reformat?

Reply:
that's what scalafmt decides it to be (format on save)